contracts/staking/interfaces/IRewardCalculator.sol

```diff
@@ -7,10 +7,11 @@ pragma solidity 0.8.25;
 
 interface IRewardCalculator {
     function calculateReward(
-        uint64 stakeAmount,
+        uint256 stakeAmount,
```

Comment:
JFYI, the P-Chain stake amounts are always represented as `uint64`s, so I originally thought it'd make sense to keep that here to help avoid the possibility of overflows in messages to/from the P-Chain.

Reply:
The P-Chain is going to have stake weights, which are going to be an arbitrary scaling of the actual stake amount. I think the contract could use either, but it make more sense to me to use the stake amount directly here